CMakeLists.txt

```diff
@@ -0,0 +1,60 @@
+cmake_minimum_required (VERSION 3.2)
+project (benchmark C)
+
+################################################################################
+# Packages
+################################################################################
+find_package(Threads REQUIRED)
+if(NOT THREADS_FOUND)
+	message(FATAL_ERROR "Threads not found")
+endif()
+
+include(GNUInstallDirs)
+
+################################################################################
+# Compiler flags:
+#   We want to use the same flags in the entire optee_client git
+################################################################################
+add_compile_options (
+	-Wall -Wextra -Werror
+#	-Wall -Wbad-function-cast -Wcast-align
+#	-Werror-implicit-function-declaration -Wextra
+#	-Wfloat-equal -Wformat-nonliteral -Wformat-security
+#	-Wformat=2 -Winit-self -Wmissing-declarations
+#	-Wmissing-format-attribute -Wmissing-include-dirs
+#	-Wmissing-noreturn -Wmissing-prototypes -Wnested-externs
+#	-Wpointer-arith -Wshadow -Wstrict-prototypes
+#	-Wswitch-default -Wunsafe-loop-optimizations
+#	-Wwrite-strings -Werror -fPIC
+# 	-Wno-missing-field-initializers
+)
+
+find_program(CCACHE_FOUND ccache)
+if(CCACHE_FOUND)
+	set_property(GLOBAL PROPERTY RULE_LAUNCH_COMPILE ccache)
+	set_property(GLOBAL PROPERTY RULE_LAUNCH_LINK ccache)
+endif(CCACHE_FOUND)
+
+set (SRC
+	main.c
+	benchmark_aux.c
+) 
+
+################################################################################
+# Built binary
+################################################################################
+add_executable (${PROJECT_NAME} ${SRC})
+
+#target_include_directories(${PROJECT_NAME} PRIVATE host/xtest)
+
+target_link_libraries (${PROJECT_NAME}
+	PRIVATE pthread
```

Comment:
same comment as in https://github.com/OP-TEE/optee_test/pull/255#discussion_r170719649